wasm/index.html

```diff
@@ -3,27 +3,42 @@
     <title>Gother World</title>
     <meta charset="utf-8"/>
     <script src="wasm_exec.js"></script>
+    <link href="https://fonts.googleapis.com/css?family=DM+Mono&display=swap" rel="stylesheet">
```

Comment:
How about using common "web safe" monospace fonts instead of using a google one? (faster page load & less tracking)

I'm thinking:
`font-family: "Monaco", "Andale Mono", "Lucida Console", "FreeMono", monospace;`

Reply:
good point! changed